src/config/ci-sim-log-store.toml

```diff
@@ -0,0 +1,10 @@
+[server]
+telemetry_enabled = false
+metrics_level = "Disabled"
+
+[system]
+max_concurrent_creating_streaming_jobs = 0
+
+[meta]
+meta_leader_lease_secs = 10
+max_heartbeat_interval_secs = 5
```

Comment:
Copied from `ci-sim`, but with this additional configuration, to force worker node to be evicted.

Reply:
If this heartbeat value was too large previously, should we set this value directly in the origin `ci-sim` and reuse the same config file so that more corner case can be tested in the integration tests and recovery tests?

Reply:
Okay, we can change it in a subsequent PR to avoid scope creep

Reply:
https://github.com/risingwavelabs/risingwave/issues/21792